bin/update

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env ruby
```

Comment:
ran `rails app:update` here. We could probably remove this but I want to look into the new `rails db:*` tasks before, I think there are some that can simplify our setup